src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,147 @@
+package duke;
+
+/**
+ * This class is responsible for parsing input from the UI, taking the
+ * String input and returning a String output.
+ */
+public class Parser {
+    private TaskList taskList;
+    private static final int DONE_SUFFIX_LENGTH = 5;
```

Comment:
Alternatively, you can define under an enum, like enum LENGTH {...}

Reply:
I'm unsure if assigning an enum for a continuous variable like length is a good idea, but I will consider it considering that the cases here are quite limited. Thank you!